### ISSUE TITLE
Activity > History - Not populating

### ISSUE THREAD

**Issue by ginjohn5snaps** [Radarr/Radarr#1252]
Nothing ever populates in my Activity > History tab. I only see the loading animation (dancing bubble thingies).

Radarr Version: 0.2.0.562 - 21 Mar 2017  Nightly (I have tried several versions - same issue)
Platform: Synology NAS

Log entries that are triggered when I see the error:
DownloadMonitoringService	Unable to retrieve queue and history items from NZBGet: Cannot access a disposed object. Object name: 'System.Net.Sockets.NetworkStream'.
NzbDroneErrorPipeline	                Request Failed: Value can not be null. Parameter name: profile
ArgumentValidator	                Value can not be null.

From what I can gather from the extremely long messages in the log entries, it is looking for something here: C:\projects\radarr-usby1\src\NzbDrone.Core\Download\TrackedDownloads\DownloadMonitoringService.cs:86 . However this place is obviously non-existent on a NAS.

Can this be fixed? It worked previously in an earlier version, but relatively lately it broke.

Issue: #1201 might be related.

**Comment by onedr0p** [comment]
Clear browser cache, try incognito window.

**Comment by ginjohn5snaps** [comment]
Saw those suggestions in previous threads yes, unfortunately clearing browser, incognito, different browser and different client did not help. I believe it is a server side issue. Is there anything I can do on the server side?

**Comment by onedr0p** [comment]
Try setting a profile on the movie, looks like it isn't set.

**Comment by ginjohn5snaps** [comment]
Double checked for profiles, and everything has it. Actually it looks like it is not possible to go without one?

I got Sonarr running alongside with the exact same configuration for everything generally, and the history works great there. Also it did work great in Radarr, but it stopped working for no apparent reason, possible following a new version. I believe it could be related to some code that looks for a file on C:/. At least that is all I gather my log.

When I started having the issue, not too long ago, I googled the subject and found an issue on here with something similar. It was apparently solved by an updated version from devs and clear cache from the user. Is it possible that it is the update that fixed that guys history that broke mine? Just a thought I had.
